server/testutil/testcache/testcache.go

```diff
@@ -66,3 +66,9 @@ func RegisterClients(env *testenv.TestEnv, conn grpc.ClientConnInterface) {
 	env.SetContentAddressableStorageClient(repb.NewContentAddressableStorageClient(conn))
 	env.SetCapabilitiesClient(repb.NewCapabilitiesClient(conn))
 }
+
+func SetupAndRun(t *testing.T, te *testenv.TestEnv) {
```

Comment:
Since this function is creating the gRPC server internally, I think it can only be used for tests that only depend on the cache, and nothing else (?)

Probably worth adding that as a comment, to avoid confusion. Or, you could remove this func for now and let tests create the gRPC server, since it's only two extra lines.